src/herder/TransactionQueue.cpp

```diff
@@ -111,8 +111,10 @@ ClassicTransactionQueue::ClassicTransactionQueue(Application& app,
     mQueueMetrics = std::make_unique<QueueMetrics>(
         sizeByAge,
         app.getMetrics().NewCounter({"herder", "pending-txs", "banned"}),
-        app.getMetrics().NewTimer({"herder", "pending-txs", "delay"}),
-        app.getMetrics().NewTimer({"herder", "pending-txs", "self-delay"}));
+        app.getMetrics().NewCounter({"herder", "pending-txs", "sum"}),
```

Comment:
forgot to rename delay-counter --> count

Reply:
Ah sorry, fixed